ci-operator/step-registry/cucushift/upgrade/arch-migration/cucushift-upgrade-arch-migration-commands.sh

```diff
@@ -182,25 +205,24 @@ function check_mcp() {
         echo "Did not run 'oc get machineconfigpools' successfully!"
         return 1
     fi
-
     # Do not check UPDATED on purpose, beause some paused mcp would not update itself until unpaused
-    oc get machineconfigpools -o custom-columns=NAME:metadata.name,CONFIG:spec.configuration.name,UPDATING:status.conditions[?\(@.type==\"Updating\"\)].status,DEGRADED:status.conditions[?\(@.type==\"Degraded\"\)].status,DEGRADEDMACHINECOUNT:status.degradedMachineCount --no-headers > "${tmp_output}" || true
+    ${OC} get machineconfigpools -o custom-columns=NAME:metadata.name,CONFIG:spec.configuration.name,UPDATING:status.conditions[?\(@.type==\"Updating\"\)].status,DEGRADED:status.conditions[?\(@.type==\"Degraded\"\)].status,DEGRADEDMACHINECOUNT:status.degradedMachineCount --no-headers > "${tmp_output}" || true
```

Comment:
`OC` is pointed to `run_command_oc` which have retry functionality, I do not think we should update it here based on these health check is stable enough, and verified for a long time.